src/main/java/cn/nukkit/entity/item/EntityPotion.java

```diff
@@ -133,14 +134,18 @@ private void splash() {
         this.getLevel().addParticle(particle);
         this.getLevel().addLevelSoundEvent(this, LevelSoundEventPacket.SOUND_GLASS);
 
-        Entity[] entities = this.getLevel().getNearbyEntities(this.getBoundingBox().grow(4.25, 2.24, 4.25));
+        Entity[] entities = this.getLevel().getNearbyEntities(this.getBoundingBox().grow(4.125, 2.125, 4.125));
         for (Entity anEntity : entities) {
             double distance = anEntity.distanceSquared(this);
-
-            if (distance < 16) {
-                double d = 1 - Math.sqrt(distance) / 4;
-
-                potion.applyPotion(anEntity, d);
+            if (anEntity.equals(collidedWith)) {
```

Comment:
Instead of two separate duplicates, we could do:
`double d = anEntity.equals(collidedWith) ? 1 : 1 - Math.sqrt(distance) / 4`

Reply:
ok

Reply:
We both forgot the semi-colon 😄